src/store/participantsStore.js

```diff
@@ -88,12 +88,14 @@ const mutations = {
 		}
 	},
 	/**
-	 * Resets the store to it's original state
+	 * Purges a given conversation from the previously added participants
 	 * @param {object} state current store state;
 	 * @param {string} token the conversation to purge;
 	 */
 	purgeParticipantsStore(state, token) {
-		Vue.delete(state.participants, token)
+		if (state.participants[token]) {
+			Vue.delete(state.participants, token)
+		} else console.debug("The conversation you are trying to purge doesn't exist")
```

Comment:
Please don't mix this if-else syntax. :)

Use proper curly braces

Reply:
Yeah, shouldnt the eslint defualt config prevent that

Reply:
I guess, I'm surprised it passed actually!